docs/can-guides/contribute/design.md

```diff
@@ -0,0 +1,11 @@
+# H1 Design Resources
+
+CanJS logos and stickers:
+
+https://drive.google.com/open?id=0B9uYesPecByGZGkzc2YxRHA4ZEE
+
+CanJS Design resources (Illustrator, Photoshop, Sketch, Etc):
+
+https://drive.google.com/open?id=0B9uYesPecByGXzhpRjlQUjBra1k
```

Comment:
I think we’ll want to keep all of the design stuff for each project in one folder, so I don’t think we need a separate Project Design Resources folder.